EIPS/eip-X.md

```diff
@@ -0,0 +1,201 @@
+* **Slashing**: a penalty applied to validators that sign _explicitly malicious_ messages that could lead to the construction and finalization of two conflicting chains (e.g. two blocks or attestations in the same slot). This penalty is designed to scale up in proportion to the number of slashable validators in the same time period such that if a critical number (wrt chain safety) of slashings have occurred, validators are _maximally_ punished.
+
+For the initial launch of Phase 0, the parameters defining the magnitude of these penalties -- `INACTIVITY_PENALTY_QUOTIENT`, `PROPORTIONAL_SLASHING_MULTIPLIER`, and `MIN_SLASHING_PENALTY_QUOTIENT` -- have been tuned to be less punitive than their final intended values. This provides a more forgiving environment for early validators and client software in an effort to encourage validation in this early, higher technical-risk stage.
+
+_`INACTIVITY_PENALTY_QUOTIENT` is four times its final value_. This results in a slower inactivity leak during times of non-finality and thus less responsiveness of the chain to such an event. If there is an extended time of non-finality during early months of eth2, it is far more likely to be technical issues with client software rather than some sort of global catastrophic event. 
+
+_`PROPORTIONAL_SLASHING_MULTIPLIER` is one-third of its final value_. This results in a lower accountable safety margin in the event of an attack. If any validators are slashed in early months of eth2, it is far more likely to be the result of user mismanagement of keys and/or issues with client software than an organized attack.
+
+_`MIN_SLASHING_PENALTY_QUOTIENT`_ is four times its final value_. This results in a lower guaranteed minimum penalty for a slashable offense and thus reduces the baseline punitive incentive to keep your system secure. As with `PROPORTIONAL_SLASHING_MULTIPLIER`, slashings during early months of eth2 are far more likely to be the result of user mismanagement or issues with client software than an organized attack.
+
+___A hard fork of the eth2 beacon chain is planned for 5 months after genesis. This hard fork will adjust `INACTIVITY_PENALTY_QUOTIENT`, `PROPORTIONAL_SLASHING_MULTIPLIER`, and `MIN_SLASHING_PENALTY_QUOTIENT` to their final, more secure values.___
+
+## Rationale
+<!--The rationale fleshes out the specification by describing what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other languages. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussion.-->
+Ethereum blocks are bursting at the seams due to increasingly high demand for decentralized applications. Ever since the first serious spikes in adoption in 2017 (cryptokitties), the Ethereum community has consistently and vocally demanded scaling solutions.
```

Comment:
```suggestion
Ethereum blocks are consistently full due to increasingly high demand for decentralized applications. Ever since the first serious spikes in adoption in 2017 (cryptokitties), the Ethereum community has consistently and vocally demanded scaling solutions.
```
Just a suggestion to be less sensational :)